ui/app/machines/tutorial-machine.js

```diff
@@ -34,7 +34,10 @@ export default {
               onEntry: { type: 'render', level: 'feature', component: 'wizard/init-setup' },
             },
             save: {
-              on: { TOUNSEAL: 'unseal' },
+              on: {
+                TOUNSEAL: 'unseal',
```

Comment:
These look slightly weird, like they should have an `_` in them, is this due to some sort of restriction, or just then way you've gone? (wheres a :seal: 🌊 emoji when you need one :joy:)

Reply:
Ah yeah this was more keeping in line with what's there already - I'll make a note to add underscores later as we've got more work to do on the on boarding stuff.